src/reducers/selectors.js

```diff
@@ -0,0 +1,114 @@
+// @flow
+
+import { createSelector } from 'reselect';
+
+export const EMPTY_OBJ = {};
+export const EMPTY_AR = [];
+
+// Checks a response object for errors and returns them
+export const processErrors = (responseValue: ?QueryResponse): Object => {
+	if (!responseValue || !responseValue.error) {
```

Comment:
I think the `processErrors` method is left over from a time ages ago where errors were nested in the `value` object and we had to process multiple requests for any possible error. Now errors match 1 to 1 with requests and the `error` isn't nested in value anymore, which leads to confusing behavior with this method. 

For example,
```js
const response = { ref: 'REF', error: 'ERROR', meta: {} };
processErrors(response); // returns `{ ref: 'REF', error: 'ERROR', meta: {} }`
processErrors({}); // returns `{}`
```
It should return just the error right? Now all we need to do is check if `error` is present and return the error. Also, the name `responseValue` is confusing now that we have distinguished between the response (ie. `resp`) and the response's value object.